lib/valkyrie/id.rb

```diff
@@ -19,7 +19,7 @@ def eql?(other)
     alias == eql?
 
     def to_uri
-      return RDF::Literal.new(id.to_s) if id.to_s.include?("://")
+      return RDF::Literal.new(id.to_s, datatype: RDF::URI("http://test.com/valkyrie_id")) if id.to_s.include?("://")
```

Comment:
Oh, sure, it's just test.com for the moment because that's what the Fedora adapter uses.